site/docs/samples/chaining-sample.md

```diff
@@ -27,20 +27,107 @@ To recreate this scenario, you will need a copy of the Fybrik repository (`git c
     ```
 
 1. Create a new namespace for the application, and set it as default:
-   ```bash
-   kubectl create namespace fybrik-airbyte-sample
-   kubectl config set-context --current --namespace=fybrik-airbyte-sample
-   ```
+    ```bash
+    kubectl create namespace fybrik-airbyte-sample
+    kubectl config set-context --current --namespace=fybrik-airbyte-sample
+    ```
+
+1. Next, register the data asset itself in the data catalog. We use port-forwarding to send asset creation requests to the OpenMetadata connector.
+    ```bash
+    kubectl port-forward svc/openmetadata-connector -n fybrik-system 8081:8080 &
+    cat << EOF | curl -X POST localhost:8081/createAsset -d @-
+    {
+      "destinationCatalogID": "openmetadata",
+      "destinationAssetID": "userdata",
+      "details": {
+        "dataFormat": "csv",
+        "connection": {
+          "name": "file",
+          "file": {
+            "connector": "airbyte/source-file",
+            "dataset_name": "userdata",
+            "format": "parquet",
+            "url": "https://github.com/Teradata/kylo/raw/master/samples/sample-data/parquet/userdata2.parquet",
+            "provider": {
+              "storage": "HTTPS"
+            }
+          }
+        }
+      },
+      "resourceMetadata": {
+        "name": "test data",
+        "geography": "theshire ",
+        "tags": {
+          "finance": "true"
+        },
+        "columns": [
+          {
+            "name": "first_name",
+            "tags": {
+              "PII": "true"
+            }
+          },
+          {
+            "name": "last_name",
+            "tags": {
+              "PII": "true"
+            }
+          },
+          {
+            "name": "birthdate",
+            "tags": {
+              "PII": "true"
+            }
+          }
+        ]
+      }
+    }
+    EOF
+    ```
+
+    The response from the OpenMetadata connector should look like this:
+    ```bash
+    {"assetID":"openmetadata-file.default.openmetadata.userdata"}
+    ```
+    Store the asset ID in a `CATALOGED_ASSET` variable:
+    ```bash
+    CATALOGED_ASSET="openmetadata-file.default.openmetadata.userdata"
+    ```
+
+    The asset is now registered in the catalog.
 
-1. Create an asset (the `userdata` asset) in fybrik's mini data catalog, the policy to access it (we use a policy that requires redactions to PII columns), and a FybrikApplication indicating the workload, context, and data requested:
+1. Create the policy to access the asset (we use a policy that requires redactions to PII columns), and a FybrikApplication indicating the workload, context, and data requested:
```

Comment:
redactions **of** PII columns ?

we create FybrikApplication in the next step. The sentence is not clear here.